portal/static/js/main.js

```diff
@@ -728,7 +728,7 @@ var fillContent = {
                                     sDisplay = "<span class='text-success small-text'>Consented / Enrolled</span>";
                                     cflag = "consented";
                             } else if (se && ir && !sr) {
-                                    sDisplay = "<span class='text-warning small-text'>Suspend Data Collection and Reports</span>";
+                                    sDisplay = "<span class='text-warning small-text'>Suspend Data Collection and Report Historic Data</span>";
```

Comment:
Paul, not sure if gettext can be applied in this context as the text is generated dynamically from client side.